### PR TITLE
Tweak settings

### PR DESCRIPTION
**Pull request by ElianHugh**
- Add vsc.dev.args setting 
- Add enum descriptions to r.source.focus
- Minor stylistic changes to item descriptions

### CODE REVIEW

File: R/session/vsc.R [hunk 1]
```diff
@@ -28,6 +28,7 @@ load_settings <- function() {
     vsc.object_timeout = session$objectTimeout,
     vsc.globalenv = session$watchGlobalEnvironment,
     vsc.plot = setting(session$viewers$viewColumn$plot, Disable = FALSE),
+    vsc.dev.args = c(width = plot$devArgs$width, height = plot$devArgs$height),
```
Comment (renkun-ken) [inline R/session/vsc.R:31]: The option is used as following:

```r
dev_args <- getOption("vsc.dev.args")
do.call(png, c(list(filename = plot_file), dev_args))
```

Maybe it makes more sense to just keep it as a list.

File: package.json [hunk 2]
```diff
@@ -1762,6 +1767,27 @@
           "default": false,
           "description": "Default boolean value for automatically sharing R browser ports with guests."
         },
+        "r.plot.devArgs" :{
+          "type": "object",
+          "markdownDescription": "The arguments for the png device to replay user graphics to show in VSCode. Requires `#r.plot.useHttpgd#` to be set to `false`. \n\nChanges the option `vsc.dev.args` in R.",
+          "default": {
+            "width": 800,
```
Comment (renkun-ken) [inline package.json:1774]: Maybe we could follow the default png width and height or at least keep its 1:1 aspect ratio as default?

```r
png(filename = "Rplot%03d.png",
    width = 480, height = 480, units = "px", pointsize = 12,
     bg = "white",  res = NA, ...,
    type = c("cairo", "cairo-png", "Xlib", "quartz"), antialias)
```

File: package.json [hunk 3]
```diff
@@ -1762,6 +1767,27 @@
...
+              "description": "Width of the graphic device.",
+              "default": 800
+            },
+            "height": {
+              "type": "number",
+              "description": "Height of the graphic device.",
+              "default": 1200
+            }
+          },
+          "additionalProperties": false
```
[the remark this answers is not in the document]
Comment (ElianHugh) [reply package.json:1789]: Ah, seeing how it leads to a call to png, that's a good point. If we remove the restriction, the settings UI will fallback to the "edit in JSON" dialog. I'm not sure if that's ideal? I think there's an argument to be made that if dots is required, people can use the R option. Alternatively, we could have '...' dots as an item for passing further args

[the remark this answers is not in the document]
Comment (renkun-ken) [reply package.json:1801]: I think this one is good enough. Let's revisit if somebody complains.

**Review by renkun-ken — APPROVED** [review]
LGTM

**Pull request merged**